src/lib/gundb/UserStorageClass.js

```diff
@@ -481,6 +481,11 @@ export class UserStorage {
           receipt,
         },
       }
+
+      if (updatedFeedEvent.type === 'bonus') {
```

Comment:
please, use constant

Reply:
@StanislavShevchenko see new commit with the changes.